python/ray/tune/tests/test_trial_scheduler.py

```diff
@@ -578,7 +568,7 @@ class _MockTrial(Trial):
     def __init__(self, i, config):
         self.trainable_name = "trial_{}".format(i)
         self.config = config
-        self.experiment_tag = "tag"
+        self.experiment_tag = "{}tag".format(i)
```

Comment:
Are there any limitations on the tag being the trial id initially? My code assumes that the tag is an integer (trial id), which can have @perturbed appended to it. I think this holds in the general use case, but not for these tests. I can change it to be more generic.